apps/ledger-live-desktop/src/renderer/components/TransactionConfirm/ConfirmTitle.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useMemo } from "react";
+import { TFunction } from "i18next";
+import { withTranslation } from "react-i18next";
+import {
+  Account,
+  AccountLike,
+  SubAccount,
+  TransactionCommon,
+  TransactionStatusCommon,
+} from "@ledgerhq/types-live";
+import { Transaction, TransactionStatus } from "@ledgerhq/live-common/generated/types";
+import Text from "~/renderer/components/Text";
+
+type Props = {
+  t: TFunction;
+  Title:
+    | React.ComponentType<{
+        account: Account | SubAccount;
+        parentAccount: Account | null | undefined;
+        transaction: TransactionCommon;
+        status: TransactionStatusCommon;
+      }>
+    | undefined;
```

Comment:
It's a bit strange to pass the ComponentType instead we should expect a `ReactNode` and we can avoid tying the props of title to this component

Reply:
Indeed that's now cleaned up following your recommendation here and the one below, same change was made for the footer

https://github.com/LedgerHQ/ledger-live/pull/6205/commits/ebc8b5f97a83710b48e6fd925a2fd8a149860c3f